css/forgetpassword.css

```diff
@@ -0,0 +1,24 @@
+
+body {
+  background:rgba (196,196,196,0.17);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5538575)